cfgov/unprocessed/css/organisms/carousel.less

```diff
@@ -0,0 +1,31 @@
+.o-carousel {
+  width: 100%;
+
+  // Override default unordered list padding.
+  & ul {
+    padding: 0;
+  }
+
+  &_item {
+    border: 1px solid red;
+    height: 100px;
+    width: 100%;
```

Comment:
Down the road, how do you envision image aspect ratios being handled? Would we maintain a fixed height, center the image in the container, and overflow the sides? i.e.

![0NrWT](https://user-images.githubusercontent.com/1060248/69167084-57ba9400-0ac2-11ea-8064-e17f3f52daa8.jpg)

Reply:
In the same vein, we'll have to reuse these images for the previews by somehow cropping them into little squares. We could use the same technique as above. Loading the full-size images in the previews would preload them for us.

Reply:
That's a good question. One direction might be to follow the approach used in https://cfpb.github.io/design-system/components/featured-content-module